backend/cancer_drug_aggregator.py

```
@@ -0,0 +1,566 @@
+    if subtype.get("type_mesh"):
+        try:
+            results = _chembl_query_with_timeout(
+                lambda: list(
+                    new_client.drug_indication.filter(mesh_id__exact=subtype["type_mesh"])
+                    .only(["molecule_chembl_id", "max_phase_for_ind", "mesh_heading", "efo_term", "mesh_id"])[:per_term_limit * 2]
+                )
+            )
+            absorb(results)
+        except FuturesTimeoutError:
+            print(f"   ⚠️  ChEMBL parent-MeSH pull timed out for {subtype['type_mesh']}")
+        except Exception as e:
```

Ruff (0.15.12)

[warning] Do not catch blind exception: `Exception`

(BLE001)